layers/+emacs/outshine/packages.el

```diff
@@ -0,0 +1,91 @@
+;;; packages.el --- outshine layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2019 Sylvain Benner & Contributors
+;;
+;; Author: Langston Barrett <langston.barrett@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `outshine-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `outshine/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `outshine/pre-init-PACKAGE' and/or
+;;   `outshine/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
+
+(defconst outshine-packages
+  '(outshine
+    outorg))
+
+(defun outshine/init-outshine ()
+  (use-package outshine
+    :defer t
+    :init
+    (progn
+      (add-hook 'prog-mode-hook          'outline-minor-mode)
+      (add-hook 'outline-minor-mode-hook 'outshine-mode))
+    :config
+    (progn
+       (spacemacs/declare-prefix "aO" "outline")
+       (spacemacs/set-leader-keys
+         "aO."  'spacemacs/outshine-transient-state/body
+         "aOS"  'outline-show-all
+         "aOgu" 'outline-up-heading
+         "aOgn" 'outline-next-heading
+         "aOgj" 'outline-forward-same-level
+         "aOgk" 'outline-backward-same-level
+         "aOih" 'outline-insert-heading
+         "aOI"  'outshine-imenu
+         "aOK"  'outline-move-subtree-up
+         "aOJ"  'outline-move-subtree-down
+         "aO>"  'outline-demote
+         "aO<"  'outline-promote))
+       (spacemacs|define-transient-state outshine
+         :title "Outshine Transient State"
+         :doc "
+     Navigate headings^^^^      Move subtrees^^                 Other^^
+     ─────────────────^^^^──    ─────────────^^───────────────  ─────^^───
+     [_j_/_k_] down/up          [_J_/_K_] move subtree down/up  [_q_] quit
+     [_n_/_N_] next/up heading  [_>_/_<_] demote/promote        [_i_] insert heading
+     [_I_] heading imenu"
+         :bindings
+         ("q" nil :exit t)
+         ("i" outline-insert-heading :exit t)
+         ("I" outshine-imenu :exit t)
+         ;; Navigate headings
+         ;; Navigate headings
+         ("n" outline-next-heading)
+         ("N" outline-up-heading)
+         ("j" outline-forward-same-level)
+         ("k" outline-backward-same-level)
+         ;; Move headings
+         ("J" outline-move-subtree-down)
+         ("K" outline-move-subtree-up)
+         ;; Move headings
+         (">" outline-demote)
+         ("<" outline-promote))))
+
+(defun outshine/init-outorg ()
+  (use-package outorg
+    :defer t
+    :config
+    (progn
+      (spacemacs/declare-prefix "aoO" "outorg")
+      (spacemacs/set-leader-keys
+        "aOe"  'outorg-edit-as-org)
```

Comment:
```suggestion
        "aOe"  'outorg-edit-as-org))))
```
Remove the following org mode key binding:
```
      (spacemacs/declare-prefix-for-mode 'org-mode "mO" "outorg")
      (spacemacs/set-leader-keys-for-major-mode 'org-mode
        "c" 'outorg-copy-edits-and-exit))))
```
Because it takes over the (possibly often used) `org-mode` binding:
`SPC m c` (or `, c` in evil mode) which calls `org-capture`.

Maybe it's enough to use the default key binding that's listed on the first line of the buffer: `*outorg-edit-buffer*`
that's opened with: `SPC a O e`
>[ packages.el ] Exit with M-#